Mage/src/main/java/mage/abilities/dynamicvalue/MultipliedValue.java

```diff
@@ -14,6 +17,9 @@ public class MultipliedValue implements DynamicValue {
     private final int multiplier;
 
     public MultipliedValue(DynamicValue value, int multiplier) {
+        if (value instanceof StaticValue) {
+            throw new IllegalArgumentException("Don't pass StaticValue into MultipliedValue, just make a StaticValue with the multiplied result");
```

Comment:
Named such errors as "Wrong code usage: xxx" and explain how to fix in error message (already has it) or in comments.